ambry-account/src/main/java/com/github/ambry/account/AbstractAccountService.java

```diff
@@ -141,11 +142,6 @@ public Account getAccountById(short id) {
               "In account " + accountName + ", container " + container.getName() + " has containerId "
                   + existingContainer.getId() + " (" + container.getId() + " was supplied)",
               AccountServiceErrorCode.ResourceConflict);
-        } else if (existingContainer.getSnapshotVersion() != container.getSnapshotVersion()) {
```

Comment:
Does this need to be changed?  I thought only account version was out of sync.

Reply:
Discussed offline.  This does need to be adjusted.  Please use the mentioned config property to toggle the behavior.